lib/blocked_edits_monitor.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class BlockedEditsMonitor
+  def self.create_alerts_for_blocked_edits(user)
+    new.create_alert(user)
+  end
+
+  def create_alert(user)
+    return if alert_already_exists?
+    alert = Alert.create!(type: 'BlockedEditsAlert',
+                          user_id: user.id,
+                          target_user_id: technical_help_staff&.id)
+    alert.email_target_user
+  end
+
+  def alert_already_exists?
```

Comment:
This method should be renamed and/or documented, to make clear that what we're checking for is whether any recent BlockedEditAlerts already exist.